pipelines/infrastructure/cloud_custodian_policies/sync_ec2_tags.yaml

```diff
@@ -0,0 +1,52 @@
+---
+policies:
```

Comment:
Probably worth creating a separate directory structure for cloud-custodian so that we can organize the policies in a way that is internally consistent to them, rather than tying them to the pipelines. If you have a different opinion on that I'm happy to be wrong.